Modules/Quotas/test/Quotas.Application.Tests/Tests/Identities/ListQuotasForIdentity/HandlerTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Backbone.BuildingBlocks.Application.Abstractions.Infrastructure.UserContext;
+using Backbone.Modules.Quotas.Application.Identities.Queries.ListQuotasForIdentity;
+using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Domain.Aggregates.Metrics;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using FakeItEasy;
+using FluentAssertions;
+using Xunit;
+using static Backbone.UnitTestTools.Data.TestDataGenerator;
+
+// ReSharper disable InconsistentNaming
```

Comment:
What's this for?

Reply:
Names such as this one bother ReSharper: `quotaGroupDTOs` - it would like `quotaGroupDtos`.

Reply:
Then just use the suggested name :)

Reply:
Done :)